src/hooks/constants/lib/fetchConstants.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiPromise } from '@polkadot/api';
+import { Constants, Fee } from '../../../generated/graphql';
+
+export const repayFeeDataType = '(u32, u32)';
+
+export const unwrapRepayFeeToFee = (apiInstance: ApiPromise): Fee => {
+  const apiRepayFee = apiInstance
+    .createType(repayFeeDataType, apiInstance.consts.lbp.getRepayFee)
+    .toHuman() as Array<string>;
```

Comment:
using `.toHuman()` is a bad idea, please explore additional means of unparsing the `Codec` received here. 

`.createType` returns a `Codec[]` which is a `ITuple<u32, u32>`, therefore you should be able to do:
```
const tuple = ... .createType(...)
tuple[0].toString() // or .toBigInt()
```

Reply:
analysis:  
- Tell me if I understand the problem+functionality we need here ? :  
  - The chain api needs to be decorated with a correct type.  
  - AND we also define the type and export it if we need it elsewhere here in this src.
- `toHuman` is bad... because it is unnecessary ?
- the `Fee.numerator/denominator` is defined/typed elsewhere as `string`
  - should we stick to that ☝️ or should we change them to `BigInt` ? 

solutions, but stated as questions,  cuz they're pretty much questions ha:
- look at the src code of our chain to see the type of `getRepayFee`,  or look at the src code of our chain to see the data structure/shape and _decide_ how to type it here ?
  - how does one find out/decide this is `Ituple<u32, u32>` ?
- define that appropriate type here in our source
- decorate the api using `createType`

😞  looks like I pretty much don't know what is going on at all here

Reply:
`.toHuman()` is bad, because it does not return programatic values, but human ones. if you have a large balance it will return `100,000,000` in a human readable format, instead of an integer. Strings can  become hexes, and vice versa. Its a bad idea overall.

We store numbers as strings on the UI, so we can use BigNumber.js with them, so no to the BigInt question.

You have to look at e.g. our testnet and look into chain state -> constants, there you can see that (u32, u32) is returned. And then if you call `.createType()` with `(u32, u32)` it gives you an `ITuple` like this:

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/6826282/154995543-2776bdd7-2490-434f-a3fd-49950f64edf5.png">

Which lets you deconstruct it like i've shown in the first comment.

Reply:
@maht0rz ready for review